git_machete/github.py

```diff
@@ -161,7 +168,7 @@ def create_pull_request(org: str, repo: str, head: str, base: str, title: str, d
     }
     prs: List[GitHubPullRequest] = derive_pull_requests(org, repo)
     to_load: GitHubPullRequest = GitHubPullRequest(1, 'user', base, head, '')
-    pr_found: Optional[GitHubPullRequest] = __check_pr_already_created(to_load, prs)
+    pr_found: Optional[GitHubPullRequest] = check_pr_already_created(to_load, prs)
```

Comment:
I'd rather change this method to sth like `check_pr_already_created_by_head_and_base`

Reply:
Also, both `check_pr_already_created...` methods could in fact be `find_pr_..`